uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.js

```diff
@@ -50,14 +50,20 @@ define([
         pending_requests: false,
         contact_id: null,
         selectedPeriod: null,
-        selectedAbsenceTypes: null
+        selectedAbsenceTypes: null,
+        assignedTo: 'all'
       }
     };
     vm.filteredUsers = [];
+    vm.filtersByAssignee = [
+      { type: 'all', label: 'All' },
+      { type: 'unassigned', label: 'Unassigned' },
+      { type: 'me', label: 'Assigned To Me' }
+    ];
     vm.isFilterExpanded = false;
+    vm.isAdmin = false; // this property is updated on controller initialization
```

Comment:
always people group single line assignments together and then multiline assignments together. you don't have to put an empty line between the two